blog/templates/blog/updateEntry.html

```diff
@@ -0,0 +1,6 @@
+{% extends "blog/blogbase.html" %}
+
+{% block blog_content %}
+    <h1>Update {{ entry.title }}</h1>
```

Comment:
I would expect to see a form on this page that would allow you to actually edit the blog entry.  It gets passed in the context with a variable named "form".